src/Marten/Events/EventMapping.cs

```diff
@@ -301,4 +301,10 @@ public override IEvent Wrap(object data)
             };
         }
     }
+
+    public static class EventMappingExtensions
```

Comment:
Is this new logic, or did you extract a method to get here?

Reply:
Yeah, I just inlined what we had above: https://github.com/JasperFx/marten/pull/2314/files#diff-8d90b88a566421c9c25b2ebb7ebfff2a6c1996c2df677e25bd9bc69392891af2L55.
I updated it to internal access.